Frontend/src/components/globalStyles/MenuBar.jsx

```diff
@@ -1,5 +1,4 @@
-import React, { useState } from 'react';
-import styles from './MenuBar.css';
```

Comment:
I don't think this was supposed to be deleted

Reply:
Added it back